03-app-instrumentation.md

```diff
@@ -115,9 +121,91 @@ Now verify the instrumentation:
 kubectl get pods -n tutorial-application -l app=backend1 -o yaml
 ```
 
+and [access traces](http://localhost:3000/grafana/explore?orgId=1&left=%7B%22datasource%22:%22tempo%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22datasource%22:%7B%22type%22:%22tempo%22,%22uid%22:%22tempo%22%7D,%22queryType%22:%22nativeSearch%22,%22serviceName%22:%22backend1-deployment%22,%22spanName%22:%22%2Frolldice%22%7D,%7B%22refId%22:%22B%22,%22datasource%22:%7B%22type%22:%22tempo%22,%22uid%22:%22tempo%22%7D,%22queryType%22:%22traceId%22%7D%5D,%22range%22:%7B%22from%22:%22now-1h%22,%22to%22:%22now%22%7D%7D)
+
+### Instrument Java - backend2 service
```

Comment:
what about the python application? Is this just an open TODO

Reply:
the auto instrumentation instructions for python are already in place